src/cli/cli_udp.cpp

```diff
@@ -174,6 +194,35 @@ otError UdpExample::ProcessSend(int argc, char *argv[])
     return error;
 }
 
+otError UdpExample::WriteCharToBuffer(otMessage *aMessage, uint16_t aMessageSize)
+{
+    otError error     = OT_ERROR_NONE;
+    int     index     = 0;
+    uint8_t character = 0x30; // 0
+
+    for (index = 0; index < static_cast<int>(aMessageSize); index++)
```

Comment:
nit: declare index as `uint16_t` to avoid `static_cast`? Also, declare `index` in the for-loop?

Reply:
yes, done.